app/main/helpers/suppliers.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+
+
+def load_countries():
+    helpers_path = os.path.abspath(os.path.dirname(__file__))
+    countryfile = os.path.join(helpers_path, '../../static/location-autocomplete-canonical-list.json')
+    with open(countryfile) as f:
+        return json.load(f)
+
+
+def get_country_name_from_country_code(country_code):
+    if country_code and country_code != 'gb':
+        potential_names = [country[0] for country in COUNTRY_LIST if country[1] == country_code]
+        return potential_names[0] if potential_names else ''
+    elif country_code:
+        # We need to support the old code style ('gb') until after the existing data has updated.
+        return 'United Kingdom'
+    else:
+        # In the case that a suppliers registration country isn't set we maintain exiting behavior.
```

Comment:
I assume this is `existing`? Also existing behaviour based on what - where is this already being done - us on the marketplace or a country picker existing behaviour thing?

Reply:
Yep existing. Currently any supplier that isn't UK based just has an empty string in the database in the registration country field.

Reply:
Think he means the typo, you currently have `exiting`

Reply:
But also might be need to detail what 'existing behaviour' is a little bit (and maybe a datestamp for quick point of reference)